docs/docs/connectors/your-own-website.mdx

```diff
@@ -155,3 +155,18 @@ as a JSON object under the key `token`:
 }
 ```
 
+### Webchat Widget
+
+Once you've set up your SocketIO channel you can use the official Rasa Webchat Widget on any webpage.
+Just paste the following into your site HTML and paste the URL of your Rasa instance into
+the `data-websocket-url` attribute
+
+```html
+<div id="rasa-webchat-widget" data-websocket-url="https://your-rasa-url-here/"></div>
+<script src="https://unpkg.com/@rasahq/rasa-webchat" type="application/javascript"></script>
+```
+
```

Comment:
Can we add something about what / how you can customize certain attributes?

Reply:
My intention was that that is covered in the next line.

Reply:
Okay, can we make that more clear? Perhaps say that it is possible and then how you would do it for one attribute as an example?